### ISSUE TITLE
[eslint-plugin-lit-a11y] aria-attr-valid-value support function calls

### ISSUE THREAD

**Issue by thepassle** [open-wc/open-wc#1880]
Seems to be a similar problem as https://github.com/open-wc/open-wc/issues/1879, see that issue for more info

Failing test:
```js
    { code: 'html`<div aria-label="${foo("foo")}"></div>`' },
```

**Comment by trystan2k** [comment]
Possible can be fixed by #1883

**Comment by thepassle** [comment]
fixed by https://github.com/open-wc/open-wc/pull/1883
